roles/kubernetes/master/templates/kubeadm-config.v1alpha3.yaml.j2

```diff
@@ -180,7 +180,6 @@ conntrack:
  tcpEstablishedTimeout: 24h0m0s
 enableProfiling: false
 healthzBindAddress: 0.0.0.0:10256
-hostnameOverride: {{ inventory_hostname }}
```

Comment:
If you remove this, then it wont be set unless you have win nodes?

Reply:
@woopstar Actually I was a bit astonished by the path of the role but couldn't see any restriction so I guess it will be set when kubeadm is enabled.